source/administration/config-settings.rst

```diff
@@ -43,6 +43,8 @@ Site URL
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The URL, including port number and protocol, from which users will access Mattermost. Leave blank to automatically configure based on incoming traffic.
```

Comment:
Yes, at least to me, *Site URL* makes it sound as if you can use something like *https://mattermost.example.com:1234/something/custom* because it's a URL too.